src/sentry/lang/native/symbolizer.py

```diff
@@ -1,14 +1,64 @@
 try:
     from symsynd.driver import Driver
     from symsynd.report import ReportSymbolizer
+    from symsynd.macho.arch import get_cpu_name
     have_symsynd = True
 except ImportError:
     have_symsynd = False
 
+from django.db import connection
 from sentry import options
 from sentry.lang.native.dsymcache import dsymcache
 
 
+def find_system_symbol(img, instruction_addr):
```

Comment:
Isn't this O(n) for number of frames? with a RT to database? Possibly two queries per frame.